src/components/Events.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from "react";
+
+export const mockEvents = [
+  {
+    id: 1,
+    name: "Vendy",
+    sport: "Basketball",
+  },
+  {
+    id: 2,
+    name: "Lukas",
+    sport: "Football",
+  },
+  {
+    id: 3,
+    name: "Honza",
+    sport: "Tennis",
+  },
+  {
+    id: 4,
+    name: "Vendy",
+    sport: "Box",
+  },
+  {
+    id: 5,
+    name: "Lukas",
+    sport: "Skydiving",
+  },
+  {
+    id: 6,
+    name: "Johana",
+    sport: "Jogging",
+  },
+];
+
+export const Events: FC = () => {
+  return (
+    <>
+      {mockEvents.map((event) => (
```

Comment:
Usually, it is a good idea in small components which won't have name collision to leverage destructuring. It helps with readability and can something tremendously help with repetition.
```suggestion
      {mockEvents.map(({ id, sport, name}) => (
```